DESCRIPTION

```diff
@@ -79,6 +79,7 @@ Suggests:
     rmarkdown,
     snowfall,
     testthat (>= 3.0.0),
+    tidyr,
     tidyverse,
```

Comment:
tidyverse maintainers have asked R package developers to _not_ add tidyverse to Imports, Depends, or Suggests in an R package (see [here](https://www.tidyverse.org/blog/2018/06/tidyverse-not-for-packages/)). The preferred approach is to list specific packages used within the tidyverse instead.

Reply:
Thanks @Andrea-Havron-NOAA for noticing this, I will remove tidyverse.

Reply:
Thanks again @Andrea-Havron-NOAA I also noticed that we needed {purrr} so I added that. And, I added a discussion about {processR} which is used in the tests but that does not need to be decided today so I am going to go ahead and merge in this PR.